romancal/skymatch/skymatch_step.py

```diff
@@ -179,7 +179,9 @@ def _set_sky_background(self, sky_image, step_status):
 
         image.meta.background.method = str(self.skymethod)
         image.meta.background.subtracted = self.subtract
-        image.meta.background.level = sky
+        # In numpy 2, the dtypes are more carefully controlled, so to match the
+        # schema the data type needs to be re-cast to float64.
+        image.meta.background.level = sky.astype(np.float64)
```

Comment:
The tighter adherence to `dtypes` means that this number should have been a `float32` before, but that violates the schemas: https://github.com/spacetelescope/rad/blob/b61c261deb1e21e13401e480a9790c6a212a0861/src/rad/resources/schemas/sky_background-1.0.0.yaml#L17. Should those be updated?